src/components/shared/card.tsx

```diff
@@ -120,24 +140,19 @@ const cardStyles = (itemType: RelatedItemType, format: Format): SerializedStyles
             return css``;
         }
 
-        case RelatedItemType.AUDIO: {
-            return css``;
-        }
-
         case RelatedItemType.LIVE: {
             return css``;
         }
 
-        case RelatedItemType.VIDEO: {
-            return css``;
-        }
-
         case RelatedItemType.REVIEW: {
             return css``;
         }
 
         case RelatedItemType.ADVERTISEMENT_FEATURE: {
-            return css``;
+            return css`
```

Comment:
Again, I think you might want this in a separate PR

Reply:
Or alternatively, update this PR to include a description of all the changes. Ideally it would be better separated but it's up to you 😄